services/src/test/scala/cromwell/services/database/QueryTimeoutSpec.scala

```diff
@@ -0,0 +1,69 @@
+package cromwell.services.database
+
+import cromwell.core.Tags.DbmsTest
+import cromwell.database.slick.{MetadataSlickDatabase, SlickDatabase}
+import org.scalactic.StringNormalizations
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.{FlatSpec, Matchers, PrivateMethodTester}
+
+import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.concurrent.duration._
+
+class QueryTimeoutSpec extends FlatSpec with Matchers with ScalaFutures with StringNormalizations with PrivateMethodTester {
```

Comment:
Pretty sure StringNormalizations isn't used, and PrivateMethodTester isn't either?